app/ui-react/syndesis/src/index.tsx

```diff
@@ -108,6 +109,11 @@ ReactDOM.render(
                                 label: 'Settings',
                                 to: routes.settings.root,
                               } as IAppRoute,
+                              {
+                                component: SupportModule,
+                                label: 'Support',
+                                to: routes.support.root,
+                              } as IAppRoute,
```

Comment:
Yeah it will render, but don’t worry. This needs to be cleaned up in a later moment, we can address this too then.